data/sql/updates/pending_db_world/rev_1630085433143335851.sql

```diff
@@ -0,0 +1,20 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1630085433143335851');
+
+SET @ITEM := 13140;
+
+-- Add Ur'dan to loot table
+UPDATE `creature_template` SET `lootid`= 14522 WHERE `entry` = 14522;
+-- Ur'dan drop rate
+DELETE FROM `creature_loot_template` WHERE `Entry` = 14522 AND `Item`= 13140;
+INSERT INTO `creature_loot_template` (`Entry`, `Item`, `Reference`, `Chance`, `QuestRequired`, `LootMode`, `GroupId`, `MinCount`, `MaxCount`, `Comment`) VALUES
+(14522, 13140, 0, 3.7, 0, 1, 0, 1, 1, 'Ur\'dan - Blood Red Key');
+-- Jaednar Darkweaver drop rate
+UPDATE `creature_loot_template` SET `Chance`= 5.6 WHERE `Entry`=7118  AND `Item`=@ITEM;
+-- Jaednar Enforcer drop rate
```

Comment:
This isn't a big deal but I think you accidentally swapped the comments here. Other than that, this looks good, thanks for making the changes!